tools/samtools.wdl

```diff
@@ -719,11 +719,11 @@ task collate_to_fastq {
     meta {
         description: "Runs `samtools collate` on the input BAM file then converts it into FASTQ(s) using `samtools fastq`"
         outputs: {
-            collated_bam: "A collated BAM (reads sharing a name next to each other, no other guarantee of sort order)"
-            read_one_fastq_gz: "Gzipped FASTQ file with 1st reads in pair"
-	        read_two_fastq_gz: "Gzipped FASTQ file with 2nd reads in pair"
-            singleton_reads_fastq_gz: "Gzipped FASTQ containing singleton reads"
-            interleaved_reads_fastq_gz: "Interleaved gzipped paired-end FASTQ"
+            collated_bam: "A collated BAM (reads sharing a name next to each other, no other guarantee of sort order)",
+            read_one_fastq_gz: "Gzipped FASTQ file with 1st reads in pair",
+	        read_two_fastq_gz: "Gzipped FASTQ file with 2nd reads in pair",
```

Comment:
Is this spacing actually off or is it just the GitHub rendering?

Reply:
Good catch. I am guessing that CoPilot inserted that tab, which it does sometimes. Very frustrating.